src/redis_zset.h

```diff
@@ -76,11 +76,13 @@ typedef struct {
   double score;
 } MemberScore;
 
-#define ZSET_INCR 1
-#define ZSET_NX (1 << 1)
-#define ZSET_XX (1 << 2)
-#define ZSET_REVERSED (1 << 3)
-#define ZSET_REMOVED 1 << 4
+enum ZSetFlags {
```

Comment:
What do you think about the naming of enumerating options? Because they aren't macro definitions now, maybe they could have naming like `MigrateTaskState` (camel case with the `k` prefix as described here https://google.github.io/styleguide/cppguide.html#Enumerator_Names)?

Reply:
done